### PR TITLE
../qa/workunits/ceph-helpers.sh: Allow TESTS to run standalone

### PR DESCRIPTION
**Pull request by wjwithagen**
Still falllout of the move to cmake building

Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

File: qa/workunits/ceph-helpers.sh [hunk 1]
```diff
@@ -17,6 +17,15 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Library Public License for more details.
 #
+if [ x"$SRC_PATH"x = xx ]; then
+  CEPH_DIR=`dirname $0`
+  CEPH_DIR=`(cd $CEPH_DIR/../..; pwd)`
```
Comment (tchaikov) [inline qa/workunits/ceph-helpers.sh]: could use `readlink` and `dirname`

File: qa/workunits/ceph-helpers.sh [hunk 2]
```diff
@@ -17,6 +17,15 @@
...
+fi
+
+CEPH_SRC=$CEPH_DIR/src
+CEPH_BIN=$CEPH_SRC
```
Comment (tchaikov) [inline qa/workunits/ceph-helpers.sh]: this is not necessarily true.

Comment (wjwithagen) [inline qa/workunits/ceph-helpers.sh]: @tchaikov 
You'd like to account for build/bin as well?

Comment (wjwithagen) [inline qa/workunits/ceph-helpers.sh]: @liewegas @tchaikov 
How desperate are we that the scripts can still be executed standalone?

I can construct `CEPH_{ROOT,BIN,LIB}` relatively easy for standalone, cmake and autobuild,
(updated the commit to match that)
but getting PYTHONPATH requires (in Cmake env):
    `./build/lib/cython_modules/lib.freebsd-11.0-CURRENT-amd64-2.7/`
to find `rados.so` 
Which is sort a far fetched path. 

I see a few solutions, but one relatively nice one would be to Cmake to actually install rados.so into something like `./build/lib` or `./build/lib/pybind`  OR replace `lib` with `bin`

**Comment by wjwithagen** [comment]
retest this please

**Comment by wjwithagen** [comment]
@liewegas @tchaikov 
This one is lingering a long time already.
Any particulars as to why it cannot/should not be merged?

**Comment by wjwithagen** [comment]
@tchaikov @liewegas 
Closing this, since it is too old. And running tests standalone on the commandline is beyond seriously broken at quite a few places.

**Pull request closed without merging**